### DISCUSSION TITLE
[TECH ED] Format data with Google Sheets.  Incorrect data shown in the video

### DISCUSSION THREAD

**Discussion by MansoorM11** [CodeYourFuture/curriculum#1278] [General]
**Describe the bug**
The data entered in column D is different from the video shown. 

**What is wrong with this content?**
<img width="1269" alt="Screenshot 2024-10-19 at 12 11 45" src="https://github.com/user-attachments/assets/b14a4b04-a7f2-46f7-8df5-aaca6753417a">

This is different from the Google Sheet we interact with. For example, the Google Sheet we use includes a dropdown menu in column D, and it contains different values, such as 'Option 1' and 'Option 2,' rather than the actual sizes of T-shirts, like 'S,' 'M,' 'L,' and 'XL.

<img width="1109" alt="Screenshot 2024-10-19 at 12 10 30" src="https://github.com/user-attachments/assets/0aae98a9-c7ce-4da0-8b77-c9b0b9433be7">

**How could this be fixed?**
The video needs to be updated or individually we need to delete the previous options from dropdown menu in column D and replace with the tee shirt sizes. 

**Link to content**

https://applieddigitalskills.withgoogle.com/c/college-and-continuing-education-uk/en-uk/g-suite-certification-sheets-part-1/g-suite-certification-sheets-part-1/sheets-duplicate-your-sheet-and-add-new-data-add-delete-and-insert-data.html

**Additional context**

**Who else might need to know about this?**

**Comment by illicitonion** [comment]
Thanks for filing this - it was a good thing to notice! It looks like on the page you linked to there is a "Report issues" button (in the top-right there's a ? button, which opens opens up a menu which shows "Report issues").

@MansoorM11 do you want to report this issue through that interface? I'm not confident it'll get fixed (re-recording a video is an annoying amount of work), but it's probably the most likely action to cause something to happen :)
